internal/cni-server/server.go

```diff
@@ -27,13 +30,27 @@ import (
 	"syscall"
 	"time"
 
+	passfd "github.com/ftrvxmtrx/fd"
 	"github.com/gorilla/mux"
 	log "github.com/sirupsen/logrus"
 
 	"github.com/merbridge/merbridge/config"
 	"github.com/merbridge/merbridge/config/constants"
+	"github.com/merbridge/merbridge/pkg/file"
 )
 
+var (
+	TransferFdSockName = "/tmp/bpf-transfer-fd.sock"
+	BpfBackServer      = "/host/var/run/bpf-back-server.sock"
+	CNITransferFdURL   = "/v1/transferFd"
+	CNIStandbyURL      = "/v1/standby"
```

Comment:
I don't see this url being called?

Reply:
This commit has not been made yet, please review the latest one. The code logic has been tested in another system and this pr may have some omissions.It should be perfect by now.